### DISCUSSION TITLE
No Access to GitHub Copilot found

### DISCUSSION THREAD

**Discussion by DLTKTejaTadepalli** [community/community#44712] [Copilot Conversations] [answered]
### Select Topic Area

Bug

### Body

Hey everyone,

I have been using the GitHub Copilot which I got under the GitHub Student Pack for a for a few months now. I read about the recent issues with GitHub Copilot and can't seem to figure out what went wrong in my case. The last email about the renewal of access to GitHub Copilot came about 10 days ago stating that the next expire date will be on 2023-02-08. Please help!

Thanks in advance.

![image](https://user-images.githubusercontent.com/91416443/213279280-cc62e5d0-f8de-41b1-aa37-066e142bc1e7.png)

#### Note:
I have tried all the possible methods stated online to try and troubleshoot this error. Still there has been no change.

**Comment by GabyCT** [comment, marked as answer]
I think this is related with https://github.com/orgs/community/discussions/43634
